lib/es-models/bib.js

```diff
@@ -841,6 +854,26 @@ class EsBib extends EsBase {
     if (typeof literal !== 'string') return null
     return [literal.substring(0, 80).toLowerCase()]
   }
+
+  /**
+   *  Get an array of subjectLiteral (or parallelSubjectLiteral) values to index
+   *
+   *  This property has very unusual value joining requirements
+   */
+  _subjectLiteralValueToIndex (primary = true) {
+    const mappings = BibMappings.get('subjectLiteral', this.bib)
```

Comment:
these two lines can be replaced with _valueToIndexFromBasicMapping

Reply:
But `EsBase._valueToIndexFromBasicMapping` returns an array of strings. I want the `VarFieldMatch` objects (so that I can work with the `subfieldMap` objects directly

Reply:
ok!